README.md

```diff
@@ -72,89 +47,136 @@ Usage
       Attachment("attachment2", Array[Byte](0, 1), "application/octet-stream", Disposition.Inline))
 ```
 
-### Sending an email synchronously
+### Sending an email asynchronously
 
 ``` scala
-  import play.modules.mailer._
-
-  val result:Try[Unit] = Mailer.sendEmail(email)
-
-  result match {
-    case Success(_) =>
-      //mail sent successfully
-    case Failure(SendEmailException(email, cause)) =>
-      //failed to send email, cause provides more information
-    case Failure(SendEmailTransportCloseException(None, cause)) =>
-        //failed to close the connection, no email was sent
-    case Failure(SendEmailTransportCloseException(Some(Success(_)), cause)) =>
-        //failed to close the connection, the email was sent
-    case Failure(SendEmailTransportCloseException(Some(Failure(SendEmailException(email, cause1))), cause2)) =>
-        //failed to close the connection, the email was not sent
-  }
+  import net.kaliber.mailer._
+
+  val mailerSettings = MailerSettings(
+    protocol = Some("smtps"),
+    host = "email-smtp.us-east-1.amazonaws.com",
+    port = "465",
+    failTo = "failto+customer@company.org",
+    auth = Some(true),
+    username = Some("Smtp username as generated by Amazon"),
+    password = Some("Smtp password")
+    
+  val result = new Mailer(Session.fromSetting(mailerSettings)).sendEmail(email)
+    
+  result
+    .map { unit =>
+      // mail sent successfully
+    }
+    .recover {
+      case SendEmailException(email, cause) =>
+      // problem sending email
+      case SendEmailTransportCloseException(result, cause) =>
+      // problem closing connection
+    }
 
 ```
 
-### Sending multiple emails synchronously
+### Sending multiple emails asynchronously
```

Comment:
One is the non-play instantiation and the other is the play instantiation version